cage.c

```diff
@@ -192,6 +192,7 @@ usage(FILE *file, const char *cage)
 		" -m extend Extend the display across all connected outputs (default)\n"
 		" -m last Use only the last connected output\n"
 		" -r\t Rotate the output 90 degrees clockwise, specify up to three times\n"
+		" -s\t Allow VT switching\n"
```

Comment:
Can someone document `-s` in manpage (since #141) as well?

Reply:
Already done in master! :smile: